python/tests/test_vmap.py

```diff
@@ -41,6 +41,8 @@ def test_unary(self):
             "log2",
             "log10",
             "logical_not",
+            "logical_or",
+            "logical_and",
```

Comment:
These are binary ops not unary ops.. I assume this test doesn't pass right now...

There is a binary test below you can move them to